src/tests/ConnectionStringTests.cs

```diff
@@ -12,11 +13,60 @@ public async Task Should_establish_connection_with_manage_rights()
         var client = new ServiceBusAdministrationClient(Environment.GetEnvironmentVariable("ASBConnectionString"));
         await client.CreateQueueAsync("testqueue");
     }
+    
+    [Test]
+    public async Task Should_have_send_claims_with_manage_rights()
+    {
+        await CreateQueueWithManageRightsIfNotExists();
+        
+        await using var client = new ServiceBusClient(Environment.GetEnvironmentVariable("ASBConnectionString"));
+        var sender = client.CreateSender("testqueue");
+        await sender.SendMessageAsync(new ServiceBusMessage(nameof(Should_have_send_claims_with_manage_rights)));
+    }
+
+    [Test]
+    public async Task Should_have_receive_claims_with_manage_rights()
+    {
+        await CreateQueueWithManageRightsIfNotExists();
+
+        await using var client = new ServiceBusClient(Environment.GetEnvironmentVariable("ASBConnectionString"));
+        var receiver = client.CreateReceiver("testqueue");
+        await receiver.ReceiveMessageAsync(TimeSpan.FromMilliseconds(500));
+    }
+    
+    async Task CreateQueueWithManageRightsIfNotExists()
+    {
+        var serviceBusAdminClient = new ServiceBusAdministrationClient(Environment.GetEnvironmentVariable("ASBConnectionString"));
+        if (!await serviceBusAdminClient.QueueExistsAsync("testqueue"))
+        {
+            await serviceBusAdminClient.CreateQueueAsync("testqueue");            
+        }
+    }
 
     [Test]
     public void Should_establish_connection_without_manage_rights()
```

Comment:
```suggestion
    public void Should_provide_connection_string_with_restricted_access()
```

or something like that?

Reply:
I would prefer to leave it this way because it uses the language of the claims which are manage, send and listen

Reply:
"Should_provide_connection_string_without_manage_rights"?

Reply:
My idea was that we want to test whether we can establish a connection using the provided connection string. That's why I framed it this way. The fact that there is a connection string available seems less important compared to it being actually usable.

Reply:
Ah, I see it now 👍